src/smc-webapp/project_log.cjsx

```diff
@@ -199,6 +199,12 @@ LogEntry = rclass
         v = if v.length > 0 then r_join(v) else 'nothing'
         <span>set <a onClick={@click_set} style={if @props.cursor then selected_item} href=''>upgrade contributions</a> to: {v}</span>
 
+    render_invite_user : ->
+        <span>invited user <User user_map={@props.user_map} account_id={@props.event.invitee_account_id} /></span>
+        
+    render_invite_nonuser : ->
+        <span>invited nonuser {@props.event.invitee_email}</span>
```

Comment:
"nonuser" doesn't sound very english to me. maybe just remove "nonuser" from that string?